src/shims/foreign_items.rs

```diff
@@ -929,6 +949,14 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
         }
         return Ok(None);
     }
+
+    fn eval_libc_i32(&mut self, name: &str) -> InterpResult<'tcx, i32> {
```

Comment:
Please add a doc comment. I had to read the code to understand what this does.^^

Also, seems like both of these `eval` functions might have a better home in `helpers.rs`?

Reply:
work in progress here: https://github.com/rust-lang/miri/pull/983